classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
+  def add_student(student)
+    @students.push(student)
+    students.classroom = self
```

Comment:
- [ ] You need to use the singular` student.classroom = self` instead of `students.classroom = self` . As you are supposed to set the classroom of the singular student. Cheers :+1: